infrastructure/src/main/java/org/corfudb/infrastructure/log/statetransfer/batchprocessor/protocolbatchprocessor/ProtocolBatchProcessor.java

```diff
@@ -51,7 +51,7 @@
      */
     @Getter
     private final ReadOptions readOptions = ReadOptions.builder()
-            .waitForHole(true)
+            .waitForHole(false)
```

Comment:
I think we need to wait for the first hole, and then later holes do not need to wait.

Reply:
Fixed